dts/riscv/it8xxx2.dtsi

```diff
@@ -240,7 +240,16 @@
 			interrupts = <39 IRQ_TYPE_EDGE_RISING>;
 			interrupt-parent = <&intc>;
 		};
-		timer: timer@f01f00 {
+
+		twd0: watchdog@f01f00 {
+			compatible = "ite,it8xxx2-watchdog";
+			reg = <0x00f01f00 0x0062>;
+			label = "TWD_0";
+			interrupts = <30 IRQ_TYPE_EDGE_RISING>;
+			interrupt-parent = <&intc>;
+		};
+
```

Comment:
- twd means "timer for watchdog"? or typo of wdt?
- It seems this register space is overlapped on next node timer@f01f10
  reg = <0x00f01f00 0x0010> is correct?

Reply:
Yes, it means "timer for watchdog".
Yes, reg address 0x00f01f00 of timer node is correct and we currently get address by DT_REG_ADDR_BY_IDX now . Once macro convert to struct, this address will also change to 0x00f01f10 in timer re-work PR.

Reply:
I understand, thanks!